docs/developer-docs/smart-contracts/maintain/settings.mdx

```diff
@@ -173,154 +98,20 @@ def get_canister_status(args: CanisterStatusArgs) -> Async[GetCanisterStatusResu
 
 There are two ways to modify the settings of a canister:
 
-1. Command line: `dfx canister update-settings <canister-id> --<field-name> <field-value>`.
-1. Programmatically: Call the `update_settings` endpoint of the IC management canister.
+1. Command line: `dfx canister update-settings <canister-name> --<field-name> <field-value>`.
+1. Programmatically: Call the [`update_settings`](/docs/current/references/ic-interface-spec#ic-update_settings) endpoint of the IC management canister.
 
 Note that only a controller of the canister can modify canister settings.
 
-An example of using `dfx` in a command line:
+### Command line
 
 ```bash
-> dfx canister update-settings ajuq4-ruaaa-aaaaa-qaaga-cai --wasm-memory-limit 3000000000
-> dfx canister status ajuq4-ruaaa-aaaaa-qaaga-cai
-...
-Wasm memory limit: 3_000_000_000 Bytes
-...
-```
-
-An example of programmatically modifying canister settings:
-
-<AdornedTabs groupId="language">
-<TabItem value="motoko" label="Motoko" default>
-
-```motoko
-actor {
-  let IC =
-    actor "aaaaa-aa" : actor {
-      update_settings : { canister_id : Principal } ->
-        async ();
-    };
-};
+dfx canister update-settings my_canister --wasm-memory-limit 3000000000
```

Comment:
Should you make it clear that this is an example or update to something like:

`dfx canister update-settings my_canister --wasm-memory-limit <limit-number>`

Reply:
I have updated this